include/z64player.h

```diff
@@ -697,7 +697,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE1_31 (1 << 31)
 
 #define PLAYER_STATE2_0 (1 << 0)
-#define PLAYER_STATE2_1 (1 << 1)
+#define PLAYER_STATE2_CAN_ACCEPT_TALK_OFFER (1 << 1) // Can talk to an actor. "Speak" or "Check" is shown on the A button.
```

Comment:
`PLAYER_STATE2_CAN_ACCEPT_TALK_OFFER`? or is there a time where player can talk without accepting a talk offer or vice versa?

Reply:
Your suggestion is the same as what I have, did you forget to type something else

Reply:
yes, sorry. `PLAYER_STATE2_CAN_TALK`?

Reply:
Yeah any time talking is initiated through c-up it is not done through the talk offer mechanism. I can update the comment to be more specific